### PR TITLE
[EM-311] Remove EAO/CEE Public Comment Period Functionality

### PR DESCRIPTION
**Pull request by jeznorth**
Adding new comment periods will no longer provide the option to go between the standard and joint comment period types.

Additional Fixes 
- Updated all context menus in the PCP interfaces to new pattern
- Fixed an issue where the spacing on the comment list
- Fixed an issue with the date picker rendering on the edit screen

### CODE REVIEW

File: modules/documents/client/views/document-manager.html [hunk 1]
```diff
@@ -165,8 +165,11 @@
                 <div class="row-actions">
                   <div class="btn-group">
 
-                    <button class="btn icon-btn dropdown-toggle" type="button" ng-click="$event.originalEvent.dropdown = true" popover-trigger="'focus'"
-                      popover-class="context-menu" uib-popover-template="'file-context-menu.html'" popover-placement="'auto'">
+                    <button class="btn icon-btn dropdown-toggle" type="button"
```
Comment (asanchezr) [inline modules/documents/client/views/document-manager.html]: @jeznorth It seems you just reformatted the `<button>` into multiple lines but the new code is missing `ng-click="$event.originalEvent.dropdown = true"` from the original code... Is that intended?

File: modules/project-comments/client/views/period-list.html [hunk 2]
```diff
@@ -58,76 +58,81 @@ <h2>Public Comment Period <span class="break">is Now Open</span></h2>
...
+                  </a>
+                </li>
+                <li ng-if="project.userCan.createCommentPeriod && p.openForComment">
+                  <a title="Delete Comment Period"
+                    ng-click="$event.stopPropagation();"
+                    x-confirm-dialog
+                    x-title-text="'Delete Comment Period'"
+                    x-ok-text="'Yes'"
+                    x-cancel-text="'No'"
+                    x-confirm-text="'This comment period is currently open for public comment, are you sure you would like to permanently delete this comment period and all its content?'"
```
Comment (NickPhura) [inline modules/project-comments/client/views/period-list.html]: The two sentences above this use "Are you sure you want to..."
This one, and the one below, use "are you sure you would like to..."

Not particularly important, but should these both use the same thing: "want to"?

File: modules/documents/client/views/document-manager.html (hunk 1 above)
Comment (jeznorth) [reply modules/documents/client/views/document-manager.html]: That ng-click event was for the previous dropdown menu implementation which is no longer required.

Comment (asanchezr) [reply modules/documents/client/views/document-manager.html]: 👍

File: modules/project-comments/client/views/period-list.html (hunk 2 above)
Comment (jeznorth) [reply modules/project-comments/client/views/period-list.html]: We can address this in another ticket to improve the technical writing. I'll review with PO and create another task if wanted.

**Pull request merged**